scantool/scantool_850/parser.py

```diff
@@ -0,0 +1,204 @@
+import sys
+import re
+
+# a class representing an OBD-II Diagnostic Trouble Code (DTC)
+class dtc:
+    # constructor
+    def __init__(self, ecu, code, prefix, suffix, description):
+        self.code = code
+        self.description = description
+        self.ecu = ecu
+        self.prefix = prefix
+        self.suffix = suffix
+
+# a class representing an ecu address and description
+class ecu:
+    # constructor
+    def __init__(self, address, description, prefix):
+        self.address = address
+        self.description = description
+        self.prefix = prefix
+    
+
+# this loads dtcs from the xiaotec file export
```

Comment:
is this function just applicable to the xiaotec import ?
Then naming should be made more uniform in comments, function names, and filenames;  currently I see inconsistent use of "Aleksi", "richard", "xiaotec", and "frobbed"